### PR TITLE
Refs #17230 - added tracer support

### PR DESCRIPTION
**Pull request by sean797**
Still waiting for `tracer-0.6.12` to be released. We can check the status from https://bodhi.fedoraproject.org/updates/?packages=tracer

### CODE REVIEW

**Comment by jlsherrill** [comment]
hah, didn't even realize this was open (but hadn't checked).  I've bumped the tag on katello-agent, so it may build now.

**Comment by jlsherrill** [comment]
[test]

File: katello-agent/katello-agent.spec [hunk 1]
```diff
@@ -33,6 +33,11 @@ Requires: yum-plugin-security
 Requires: yum-security
 %endif
 
+%if 0%{?fedora} > 18 || 0%{?rhel} > 6
+Requires: python2-tracer >= 1.6.12 
+Requires: python-rhsm
+%endif
```
Comment (jlsherrill) [inline katello-agent/katello-agent.spec:39]: curious if you had thought about making this a subpackage that is only built on those platforms?

**Comment by jlsherrill** [comment]
ah right, you'll need to download https://github.com/Katello/katello-agent/archive/2.8.0.tar.gz and add it via git annex add 2.8.0  (named as 2.8.0 due to the silly naming that github gives us via redirection)

File: katello-agent/katello-agent.spec (hunk 1 above)
Comment (sean797) [reply katello-agent/katello-agent.spec:39]: Create a katello-agent-tracer pkg? I guess we could do but no I hadn't thought about it.

**Comment by jlsherrill** [comment]
ACK thanks @sean797 !

**Pull request merged**